.eslintrc.js

```diff
@@ -1,139 +1,94 @@
-    ],
-    // TODO: Set 'variables' to 'true' after fixing the existing issues
-    'no-use-before-define': ['warn', { 'functions': true, 'classes': true, 'variables': false }],
-    'no-useless-rename': 'warn',
-    'no-var': 'warn',
-    'object-shorthand': 'warn',
-    'prefer-arrow-callback': 'warn',
-    'prefer-const': 'warn',
-    'prefer-template': 'warn',
-    'prefer-rest-params': 'warn',
-    'prefer-spread': 'warn',
-    'rest-spread-spacing': 'warn',
-    'template-curly-spacing': 'warn',
-    'vars-on-top': 'warn',
-
+  env: {
+    jest: true,
+  },
+  rules: {
+    indent: 'off',
+    'comma-dangle': 'off',
+    'import/prefer-default-export': 'off',
 
-    'lodash-fp/consistent-name': ['warn', '_'],
-    'lodash-fp/no-argumentless-calls': 'warn',
-    'lodash-fp/no-chain': 'warn',
-    'lodash-fp/no-extraneous-args': 'warn',
-    'lodash-fp/no-extraneous-function-wrapping': 'warn',
-    'lodash-fp/no-extraneous-iteratee-args': 'warn',
-    // 'lodash-fp/no-extraneous-partials': 'warn', // available in eslint-plugin-lodash-fp 2.2.0+
-    'lodash-fp/no-partial-of-curried': 'warn',
-    'lodash-fp/no-single-composition': 'warn',
-    'lodash-fp/no-submodule-destructuring': 'warn',
-    'lodash-fp/no-unused-result': 'warn',
-    'lodash-fp/prefer-compact': 'warn',
-    'lodash-fp/prefer-composition-grouping': 'warn',
-    'lodash-fp/prefer-flat-map': 'warn',
-    'lodash-fp/prefer-get': 'warn',
-    'lodash-fp/prefer-identity': ['warn', { 'arrowFunctions': false }],
-    'lodash-fp/preferred-alias': 'warn',
-    'lodash-fp/use-fp': 'warn',
+    // TODO: ticket all of these to be fixed or considered
+    'react/forbid-prop-types': 'off',
+    'react/prop-types': 'off',
+    'react/sort-comp': 'off',
+    'react/destructuring-assignment': 'off',
+    'react-hooks/exhaustive-deps': 'off',
+    'react/require-default-props': 'off',
+    'react/static-property-placement': 'off',
 
+    'import/no-cycle': 'off',
+    'import/no-named-as-default': 'off',
+    'import/no-extraneous-dependencies': 'off',
+    'import/extensions': 'off',
+    'import/no-relative-parent-imports': 'off',
 
-    'react/default-props-match-prop-types': 'warn',
-    'react/no-children-prop': 'warn',
-    'react/no-unused-state': 'warn',
+    '@typescript-eslint/default-param-last': 'off',
+    '@typescript-eslint/return-await': 'off',
+    '@typescript-eslint/no-shadow': 'off',
+    '@typescript-eslint/no-use-before-define': 'off',
+    '@typescript-eslint/no-unused-expressions': 'off',
+    '@typescript-eslint/strict-boolean-expressions': 'off', // TODO: should be 'warn',
+    '@typescript-eslint/no-implied-eval': 'off',
+    '@typescript-eslint/no-throw-literal': 'off',
+    '@typescript-eslint/naming-convention': 'off',
 
-    'react-hooks/rules-of-hooks': 'warn',
-    'react-hooks/exhaustive-deps': 'warn',
+    'no-case-declarations': 'off',
+    'no-nested-ternary': 'off',
+    'no-console': 'off',
```

Comment:
The current configuration allows only assert and error, not all console logs. Can we keep that?